detekt/distribution.bzl

```diff
@@ -0,0 +1,20 @@
+"""
+A macro for changing Detekt distribution.
+"""
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+def detekt_distribution(version = "1.2.2"):
+    maven_install(
```

Comment:
I'm not exactly sure what's the story with overriding maven_install inflated from other WORKSPACE, do you want to try to to override detekt to a custom jar to test if that's actually possible?